nixos/modules/services/web-servers/meguca.nix

```diff
@@ -1,65 +1,72 @@
 { config, lib, pkgs, ... }:
 
-with lib;
 let
   cfg = config.services.meguca;
   postgres = config.services.postgresql;
-in
-{
+in with lib; {
   options.services.meguca = {
     enable = mkEnableOption "meguca";
 
-    baseDir = mkOption {
+    dataDir = mkOption {
       type = types.path;
-      default = "/run/meguca";
+      default = "/var/lib/meguca";
```

Comment:
Changing the default will break everybody's setup that hasn't set the option explicitly.

Reply:
Oh haha, I see, never mind then